gqcp/include/QCModel/HF/RHF.hpp

```diff
@@ -447,7 +451,7 @@ class RHF {
             for (const auto& a : orbital_space.indices(OccupationType::k_virtual)) {
                 for (const auto& j : orbital_space.indices(OccupationType::k_occupied)) {
                     for (const auto& b : orbital_space.indices(OccupationType::k_virtual)) {
-                        singlet_A_slice(i, a, j, b) = 2 * g(a, i, j, b) - g(a, b, j, i);
+                        singlet_A_slice(i, a, j, b) = g(a, i, j, b) + g(a, i, j, b) - g(a, b, j, i);
```

Comment:
I fail to see why the new implementation produces correct results, and the old one doesn't. Both formulas are exactly the same.

Reply:
This was done because the old implementation resulted in an error when trying to bind the complex stability methods. The operation `*` was not compatible between a `complex double` and an `integer`. I found this really strange myself, but since it was an issue of type compatibility, this fixed it. 

I'm not a big fan of it either, so if you have any other suggestions to fix this, I'd be happy to change it. But as long as the complex bindings are enabled, `2 * g(a, i, j, b)` won't compile.

Reply:
![Screenshot 2020-12-07 at 13 41 25](https://user-images.githubusercontent.com/54795591/101352349-480b1a80-3892-11eb-9443-b1e0a45d54ba.png)

This is a screenshot of the error when `2 * g(a, i, j, b)` is used.

Reply:
Oh, I see. I think you can use `2.0` (as a literal `double`) and then it will compile.

On another note. Have you double-checked the correctness of these formulas in the complex case? (From experience, I know that a prefactor `2` in the real case often results from the permutational symmetry of **real** integrals.)

Reply:
You were correct, using a factor `2.0` did the trick!

As for the correctness, the factor 2 comes directly from the article from Pople, where the complex case is tackled as well as the real case.